src/NoteBook.js

```diff
@@ -0,0 +1,22 @@
+class Notebook {
+
+  constructor(listOfNotes) {
+    this.listOfNotes = ['I need to go and buy a football', 'I want to go bowling', 'go and code'];
+  
+  }
+
+  truncatedNotes(){
+    for (const noteIndex of this.listOfNotes) {
+      if(noteIndex.length > 20) { 
+        return noteIndex.substring(0, 20)+'...'
+      }else if(noteIndex.length <= 20) {
+        return noteIndex
+      }
+    }
+  }
```

Comment:
This will hit return immediately for the very first note in the list. So instead of returning an array of all the notes truncated, it will just return a string of the first truncated note and nothing else